src/cmake/externalpackages.cmake

```diff
@@ -107,14 +121,14 @@ else ()
     if (NOT USE_STD_REGEX)
         list (APPEND Boost_COMPONENTS regex)
     endif ()
-    find_package (Boost 1.53 REQUIRED
-                  COMPONENTS ${Boost_COMPONENTS})
+    find_package (Boost REQUIRED COMPONENTS ${Boost_COMPONENTS})
```

Comment:
What's up with this change? Don't we need that "1.53" to enforce the minimum Boost version we intend?